x/emissions/keeper/topic_weight.go

```diff
@@ -45,16 +45,13 @@ func (k *Keeper) GetCurrentTopicWeight(
 	topicRewardAlpha alloraMath.Dec,
 	stakeImportance alloraMath.Dec,
 	feeImportance alloraMath.Dec,
-	additionalRevenue cosmosMath.Int,
-	additionalStake cosmosMath.Int,
 ) (weight alloraMath.Dec, topicRevenue cosmosMath.Int, err error) {
 	topicStake, err := k.GetTopicStake(ctx, topicId)
 	if err != nil {
 		return alloraMath.Dec{}, cosmosMath.Int{}, errors.Wrapf(err, "failed to get topic stake")
 	}
 
-	newTopicStake := topicStake.Add(additionalStake)
```

Comment:
I recall there being reasons why I had to add this topicStakeDec line like this

I am not comfortable deleting it